app/lib/services/wallet_service.dart

```diff
@@ -70,7 +70,8 @@ Future<List<Wallet>> listWallets() async {
     return await Future.wait(walletFutures);
   }, null);
 
-  return wallets;
+  final uniqueWallets = wallets.toSet().toList();
```

Comment:
I think we did this solution before here:
https://github.com/threefoldtech/threefold_connect/blob/411ffc408683f59d512d4c1d56eb07c53a562587/app/lib/services/wallet_service.dart#L56

Maybe we need to know the root cause of why the daily wallet is being duplicated and fix it instead.

Reply:
- I checked the solution that was done as u mentioned. It's done but it wasn't working because of the missing override in the pkIdWallet class.
- so i moved the override to  pkIdWallet and now the existing solution works.
- as agreed with @xmonader  to keep this solution and open separate issue for further investigation about why the daily wallet may get duplicated.